oidc-ui/src/common/ModalPopup.js

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+
+const ModalPopup = ({alertIcon, header, body, footer}) => {
+  return (
+    <div
+      class="relative z-10"
+      aria-labelledby="modal-title"
+      role="dialog"
+      aria-modal="true"
+    >
+      <div class="fixed inset-0 bg-gray-500 bg-opacity-75 transition-opacity"></div>
```

Comment:
Is this div is necessary, if not, you can remove this div

Reply:
Yes, it is required to have a blur background when the popup appears.